nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/ConanProxyHelper.groovy

```diff
@@ -1,12 +1,15 @@
 /*
  * Sonatype Nexus (TM) Open Source Version
  * Copyright (c) 2017-present Sonatype, Inc.
- * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype
```

Comment:
exclude not VCS code from auto reformatting

Reply:
fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/97/commits/d7566f3ca7a5297d9e90aac735efefebc064b31d